build.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+# Correct Syntax: ./build.sh [COM port] [verify/build] 
```

Comment:
Usually pipe is used to denote one of several options.  See `git help reset` for an example.

I think it's good to use `set -e` and `set -u` at the top of every shell script, which will exit the script if any command fails and if you try to use an undefined variable, respectively.